lib/cosmos/tools/tlm_grapher/data_object_adders/housekeeping_data_object_adder.rb

```diff
@@ -64,9 +71,10 @@ def update
     end
 
     # Adds a data object when the add button is pressed
-    def add_data_object(target_name, packet_name, item_name)
+    def add_data_object(target_name, packet_name, item_name, item_array_index)
       data_object = HousekeepingDataObject.new
       data_object.set_item(target_name, packet_name, item_name)
+      data_object.item_array_index = item_array_index
       @add_data_object_callback.call(data_object) if @add_data_object_callback
```

Comment:
The adder doesn't work if select an array item without doing [0] etc.  It should work and default to the first item in the array